src/app/pages/send/send-crypto-asset-form/form/btc/use-btc-send-form.tsx

```diff
@@ -57,23 +57,35 @@ export function useBtcSendForm() {
       setIsSendingMax(value);
     },
     utxos,
+    /** #3766 - indavertant error display
+     * - we show validation errors on focus and re-entry before changing value
+     * - $other allows us to add a non empty check without cylical dependancy
+     * - we then only run validation if the amount is not empty
+     */
```

Comment:
This solves this issue on `amount` but maybe:
- there's a cleaner way to solve it here and everywhere?
- I should apply this logic in a different place so it will fix the issue on multiple fields?

Reply:
Thanks Kyran, yes that may be cleaner so I will look into that.

It seems once you enter and leave a field it will show the first `.test({ message: '` no matter what. 

It could be good to only validate `onChange` rather than `onBlur` but I can look at the forms we have and make an assessment.

Reply:
Also, I made the hook `useIsFieldDirty` recently, so you can focus/blur without revealing error if the value stays empty. I wonder if we need something else in addition to this. Why is it exactly the wrong validation error is shown on first edits?

Reply:
I was debugging those hooks last week and what happens is:
- once a field is touched and `meta.touched` becomes `true`
- the first validation message in the list for that field is set as `meta.error`
- then when a user starts typing the field is considered `isDirty` 
- this means the original `meta.error` will show until `onBlur` runs again and the validation runs

It could be simpler to add a fix in the hooks but I did try that without luck yet. I will dig deeper into this and try and come up with a better fix for all fields.

Usually with `onChange` validation, there would be a debounce time so that it won't run validation until a few seconds later to allow the user to stop typing.

I notice also that all fields are treated as type `text` so users can type in letters to the amount for example. We can consider tightening this up to restrict them to `number` here for example, as a form of pre-validation

Reply:
Thanks for explanation, makes sense. 

Would be nice if there was a generic, reusable way we could set the logic that: if onBlur, fieldValue === initialValue, set touched to false, but formik seems to have little configuration available. 

Feel free to update to correct input type. Maybe related, [some UX issues with `input type=number`](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/)